app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerActivity.kt

```diff
@@ -27,9 +28,9 @@ class ServerActivity : DaggerAppCompatActivity() {
         viewModel.shouldDrawerBeOpen.observe(this, Observer { shouldClose ->
             shouldClose ?: return@Observer
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/3c5401c2c1e764397645482fac20357348df064f/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ServerActivity.kt#L29"></span>Expression with labels increase complexity and affect maintainability.</td>
    </tr>
  </tbody>
</table>